_html/scss/modules/_homepage-burger-menu.scss

```diff
@@ -0,0 +1,136 @@
+.header--menu,
+.burger-button {
+    display: inline-block;
+    vertical-align: middle;
+}
+
+.link-menu {
+    right: 6rem;
+}
+
+.burger-button {
+    width: 25px;
+    height: 25px;
+    margin-left: 20px;
+    cursor: pointer;
+    z-index: 10;
+
+    .btn-layers {
+        @include prefix(transform, translateY(-50%), webkit moz ms o);
```

Comment:
You don't need to add prefixes since they are added by Gulp. Delete this and the mixin.